readme.md

```diff
@@ -40,8 +40,8 @@ Specify your own:
 
 ###### `options.comment`
 
-Character to use for line-comments, `false` turns off comments (`string` or
-`boolean`, default: `'%'`)
+Character or a characters array to use for line-comments, `false`
+turns off comments (`string` or`boolean`, default: `'%'`)
```

Comment:
```suggestion
Character(s) to use for line-comments, `false` turns off comments (`string`,
`Array.<string>`, or `boolean`, default: `'%'`)
```

Reply:
Sure!
I don't speak English very well, I did the best I could.